docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -106,83 +104,79 @@ Ethereum operates as a single, homogeneous chain. Each Ethereum node is divided
+[GHOST](https://www.geeksforgeeks.org/what-is-ghost-protocol-for-ethereum/)-based and can both
+finalize batches of blocks in one round.
+
+For block production, both protocols use slot-based protocols that randomly assign validators to a
+slot and provide a fork choice rule for unfinalized blocks. Polkadot uses
+[BABE](./learn-consensus.md#badass-babe-sassafras) for block production. BABE includes two
+mechanisms for selecting block producers, one of which is a fallback in case the first fails, which
+allows for chain liveness. BABE produces unfinalized blocks on top of the chain already finalized by
+GRANDPA.
 
 There are two main differences between Ethereum and
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} consensus:
 
-1.  Ethereum finalizes batches of blocks according to periods called "epochs". The current plan is
-    to have 32 blocks per epoch and finalize them all in one round. With a predicted block time of
-    12 seconds, the expected time to finality is 6 minutes (12 minutes maximum). See
-    [Ethereum 2 Block Time](https://github.com/ethereum/eth2.0-specs/blob/676e216/specs/phase0/beacon-chain.md#time-parameters)
-    for more information.
-
-{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} finality protocol, GRANDPA,
-finalizes batches of blocks based on
-[availability and validity checks](./learn-parachains-protocol.md#availability-and-unavailability-phase)
-that happen as the proposed chain grows. The time to finality varies with the number of checks that
-need to be performed (and invalidity reports cause the protocol to require extra checks). The
-expected time to finality is 12-60 seconds.
-
-2.  Ethereum requires many validators per shard to provide strong validity guarantees while
-    {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} can provide stronger guarantees
-    with fewer validators per shard. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
-    achieves this by making validators distribute an
-    [erasure coding](./learn-parachains-protocol.md#erasure-codes) to all validators in the system,
-    such that anyone - not only the shard's validators - can reconstruct a parachain's block and
-    test its validity. The random parachain-validator assignments and secondary checks are performed
-    by randomly selected validators making it less likely for the small set of validators on each
-    parachain to collude.
-
-### Staking Mechanics
+1. {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} finality protocol, GRANDPA,
+   finalizes batches of blocks based on
+   [availability and validity checks](./learn-parachains-protocol.md#availability-and-unavailability-phase)
+   that happen as the proposed chain grows. The time to finality varies with the number of checks
+   that need to be performed (and invalidity reports cause the protocol to require extra checks).
+   The expected time to finality is 12-60 seconds.
```

Comment:
isn't it 30 seconds? We are mentioning this is the async backing comparison table. We should be consistent.